### PR TITLE
Update radare2

### PR DESCRIPTION
**Pull request by pelijah**
**Test plan (required)**

**Closing issues**

### CODE REVIEW

**Comment by a1ext** [comment]
Related to #1179

**Comment by pelijah** [comment]
изображение

File: src/core/Cutter.cpp [hunk 1]
```diff
@@ -393,7 +394,7 @@ bool CutterCore::loadFile(QString path, ut64 baddr, ut64 mapaddr, int perms, int
     }
 
     ut64 hashLimit = getConfigut64("cfg.hashlimit");
-    r_bin_file_hash(core_->bin, hashLimit, path.toUtf8().constData());
+    r_bin_file_hash(core_->bin, hashLimit, path.toUtf8().constData(), nullptr);
```
Comment (xarkes) [inline src/core/Cutter.cpp]: I'm unsure but I'd say it's best to give `NULL` to the C APi rather than `nullptr`.

File: src/core/Cutter.cpp [hunk 2]
```diff
@@ -784,7 +785,7 @@ QString CutterCore::assemble(const QString &code)
 QString CutterCore::disassemble(const QString &hex)
 {
     CORE_LOCK();
-    RAsmCode *ac = r_asm_mdisassemble_hexstr(core_->assembler, hex.toUtf8().constData());
+    RAsmCode *ac = r_asm_mdisassemble_hexstr(core_->assembler, nullptr, hex.toUtf8().constData());
```
Comment (xarkes) [inline src/core/Cutter.cpp]: Same wonder here.

**Pull request merged**